src/main/java/thrift/logic/Logic.java

```diff
@@ -22,7 +23,8 @@
      * @throws CommandException If an error occurs during command execution.
```

Comment:
Remember to add javadocs for the new @param

Reply:
Thanks for reminding!